build.gradle

```diff
@@ -623,9 +623,9 @@ task testUtilsJavadocJar(type: Jar, dependsOn: testUtilsJavadoc){
     from "$docBuildDir/testUtilsJavadoc"
 }
 
-task testUtilsSourceJar(type: Jar){
+task testUtilsSourcesJar(type: Jar){
     baseName = "$project.name-test-utils"
-    classifier = 'source'
+    classifier = 'sources'
```

Comment:
Wow - was this just silently not working ?

Reply:
It was loudly not working when we tried to do a maven release.